src/content/docs/infrastructure/infrastructure-monitoring/get-started/choose-infra-install-method.mdx

```diff
@@ -0,0 +1,89 @@
+
+* [Install on Linux systems with a package manager](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux)
+* [Install on macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos)
+* Install on Windows with [MSI install](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows), [zip install](/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows), or [manual zip install](/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-manual-install-infrastructure-agent-windows).
+
+## Deploy the infrastructure agent programmatically [#deploy-programmatically]
+
+You can deploy the infrastructure agent programmatically with several config management and deploy tools:
+
+    * <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="Ansible" alt="Ansible" src={osAnsibleRed}/> [Ansible](/docs/infrastructure/new-relic-infrastructure/config-management-tools/configure-new-relic-infrastructure-using-ansible)
+    * <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="Chef" alt="Chef" src={osChef}/> [Chef](/docs/infrastructure/new-relic-infrastructure/config-management-tools/configure-new-relic-infrastructure-using-chef)
+    * <img style={{ width: '32px', height: '28px', verticalAlign: 'middle'}} class="inline" title="Docker" alt="Docker" src={osDocker}/> [Docker](/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring) (install as container)
+    * <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="EBS" alt="EBS" src={osEbs}/>[Elastic Beanstalk](/docs/infrastructure/install-infrastructure-agent/config-management-tools/install-infrastructure-agent-aws-elastic-beanstalk)
+    * <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="Puppet" alt="Puppet" src={osPuppet}/> [Puppet](/docs/infrastructure/new-relic-infrastructure/config-management-tools/configure-new-relic-infrastructure-puppet)
+
+## Use the guided install [#guided-install]
+
+The guided install is a single CLI command you can run to monitor your instance. It's a good option for small organizations, or for anyone who wants to test out New Relic. When you run the CLI command, New Relic will inspect your system to find other compatibilty technologies that you may want to monitor. You'll also receive the [infrastructure status API](https://github.com/newrelic/infrastructure-agent/blob/master/docs/status_api.md) by default.
```

Comment:
instance is almost certainly too vague here

Reply:
tbh this is just boilerplate language i pulled from across the docs for the guided install. im going to keep it for consistency because it doesn't install the agents into your environment, just an instance of a host (or app, or whatever)